recipes-support/collision-avoidance-library/coav-control_0.0.1.bb

```diff
@@ -24,6 +24,6 @@ do_install () {
     chrpath -d ${B}/tools/coav-control/coav-control
```

Comment:
why are we not just installing the binary by "make install"  rather than doing this?